h/zos.h

```diff
@@ -1529,6 +1529,10 @@ DSAB *getDSAB(char *ddname);
 
 int dsabIsOMVS(DSAB *dsab);
 
+void message2(char *message);
+
+void wtoPrintf3(char *formatString, ...);
```

Comment:
Why 3 and not 2?

Reply:
wtoPrintf2 already exists @ifakhrutdinov